### ISSUE TITLE
Nimbus LAF: Menu -> Team -> Show history renders wrong with non default font

### ISSUE THREAD

**Issue by PavelTurk** [apache/netbeans#4272]
### Apache NetBeans version

Apache NetBeans 14

### What happened

I need a font size bigger then a default one, so, in netbeans.conf I add the following `--fontsize 16 --laf Nimbus -J-Dswing.aatext=true -J-Dawt.useSystemAAFontSettings=lcd`. However, with such settings git history renders bad:

![Screenshot from 2022-06-22 19-29-19](https://user-images.githubusercontent.com/10261361/175086793-0fe275ac-eb89-434c-a91e-adda0f47b028.png)

### Did this work correctly in an earlier version?

No

### Operating System

Ubuntu 20.04.3 LTS

### JDK

17.0.2; OpenJDK 64-Bit Server VM 17.0.2+8-86

### Apache NetBeans packaging

Apache NetBeans provided installer

### Anything else

At the same, it is strange that sometimes it is rendered correctly, but most of the time wrongly.

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by mbien** [comment]
how does this look with default FlatLAF but with increased font size?

**Comment by mbien** [comment]
I don't think that they are linked since the font is large in this case, the component just doesn't resize to fit the text. The other issue you filed was about the font sometimes *not* being large. Btw did you try the suggestion posted there?

Nimbus is a fairly old LAF which never had the quality of the other LAFs. Additionally, NB has many custom components which won't render properly unless they are optimized for that LAF. 

You don't like FlatLAF? Since that is the default and the main supported LAF.

**Comment by mbien** [comment]
@PashaTurok I don't know but we can ask someone who might know @neilcsmith-net @eirikbakke @DevCharly

(I'm personally using "FlatLaf Dark")

**Comment by eirikbakke** [comment]
> I don't like FlatLAT. It is too light for me.

@PashaTurok I have a few improvements to FlatLAF pending at https://github.com/apache/netbeans/pull/3115 , which were done in particular to improve the appearance of tab components in the window system. Would that patch meet your needs? (See the before/after screenshots.)

**Comment by neilcsmith-net** [comment]
@mbien well there was some discussion a while back on supporting FlatLaf themes - ie. https://www.formdev.com/flatlaf/themes/

**Comment by neilcsmith-net** [comment]
@PashaTurok FYI from NetBeans 16 you can override any property in FlatLaf by editing the custom properties file from `Tools / Options / Appearance / FlatLaf`.  Might help your situation.
